two-container/docker-compose.yaml

```diff
@@ -38,6 +38,8 @@ services:
     ports:
       - 53/tcp
       - 53/udp
+    volumes: 
+      - /volume1/docker/pihole-unbound/unbound:/opt/unbound/etc/unbound/
```

Comment:
Hard-coded path -- assumes that the path to your configs is `/volume1/docker/pihole-unbound/`, not a safe assumption.